phoenix-core/src/test/java/org/apache/phoenix/query/BaseTest.java

```diff
@@ -629,6 +626,11 @@ public static Configuration setUpConfigForMiniCluster(Configuration conf, ReadOn
         conf.setInt(HConstants.HBASE_CLIENT_RETRIES_NUMBER, 2);
         conf.setInt(NUM_CONCURRENT_INDEX_WRITER_THREADS_CONF_KEY, 1);
         conf.setInt(GLOBAL_INDEX_ROW_AGE_THRESHOLD_TO_DELETE_MS_ATTRIB, 0);
+        if (conf.getLong(QueryServices.UNGROUPED_AGGREGATE_PAGE_SIZE_IN_MS, 0) == 0) {
+            conf.setLong(QueryServices.UNGROUPED_AGGREGATE_PAGE_SIZE_IN_MS, 0);
```

Comment:
Did you mean to set this to some non-zero value?

Reply:
I set it to 1 ms first but observed that even within 1ms pages, several iterations can be processed in a page . So, in order to make sure that every test is subject to paging, I had to set it to 0, which results in one-row pages.

Reply:
I meant, the `if` condition is checking that the config is 0 and then setting it to zero itself. Perhaps you meant to invert the condition

Reply:
Oh nevermind. I see that you're passing in 0 as the default in the getLong.